third_party/accessibility/ax/ax_node.h

```diff
@@ -120,6 +120,7 @@ class AX_EXPORT AXNode final {
   size_t GetUnignoredChildCount() const;
   AXNode* GetUnignoredChildAtIndex(size_t index) const;
   AXNode* GetUnignoredParent() const;
+  OwnerTree::Selection GetUnignoredSelection() const;
```

Comment:
Can we keep Chromium's comment?

```suggestion
// Gets the unignored selection from the accessibility tree, meaning the
  // selection whose endpoints are on unignored nodes. (An "ignored" node is a
  // node that is not exposed to platform APIs: See `IsIgnored`.)
  OwnerTree::Selection GetUnignoredSelection() const;
```